tool-openssl/c_rehash.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+# Get absolute path to script directory
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd)
+
+# Check for openssl binary in same directory
+if [ -f "${SCRIPT_DIR}/openssl" ]; then
```

Comment:
This should check that the file is executable.

```
if [ -x "${SCRIPT_DIR}/openssl" ]; then
```

Reply:
Right, the question is whether we want our custom error message to be displayed in this case. I suppose it's highly unlikely that the file would exist but not be executable.